src/layouts/AllServices.tsx

```diff
@@ -0,0 +1,542 @@
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>
+                        Streamline your hybrid cloud experience, reducing the operational cost and complexity of delivering cloud-native applications.
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/beta/application-services/api-designer">
+                          API Designer
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/application-services/api-management">
+                          API Management
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/beta/application-services/connectors ">
+                          Connectors
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/application-services/service-accounts ">
+                          Service Accounts
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/application-services/service-registry ">
+                          Service Registry
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/application-services/streams/overview ">
+                          Streams of Apache Kafka
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </CardBody>
+                </Card>
+                <Card isPlain>
+                  <CardTitle>
+                    <AutomationIcon /> Automation
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>Solve problems once, in one place, and scale up. </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/ansible/automation-analytics/reports">
+                          Automation Analytics
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/ansible/automation-hub">
+                          Automation Hub
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
```

Comment:
Instead of `href="#"` we should add `onClick={(e) => e.preventDefault()}` that way we don't direct user to empty anchor. Looking at this specific item, shouldn't this navigate user to insights dashboard?